src/components/TabsPage/TabsPage.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { memo, useMemo } from 'react';
+import { useParams } from 'react-router-dom';
+import { tabs } from '../../api/Tabs';
+import { TabItem } from '../TabItem/TabItem';
+
+export const TabsPage: React.FC = memo(() => {
+  const { tabId } = useParams();
+  const selectedTab = useMemo(() => tabs.find(
+    tab => tab.id === tabId,
+  ), [tabs, tabId]);
+
+  return (
+    <div className="section">
+      <div className="container">
```

Comment:
```suggestion
    <div className="section">
      <div className="container">
```

this two divs has to be in App file, so then you will have same formatting for both pages. Now you have wrong margins in your Home page.